### ISSUE TITLE
3318494 Max Init Code Size issue

### ISSUE THREAD

**Issue by letypequividelespoubelles** [Consensys/besu-sequencer-plugins#1138]
## Block numbers

```rust
3318494-3318494
```

## Issue

3318494 - SEPOLIA with 0.5.3-beta

```
Invalid transaction: Initcode size of 34919 exceeds maximum size of 24576
```
But the transaction doesn't fail IRL https://explorer.sepolia.linea.build/block/0x3fe1b976141ef7a18f5d5c10ffd97726c224e7e4dbc4adeb5d54628e11585681?tab=txs

**Comment by DavePearce** [comment]
Looking at the initcode declared on sepolia, it does indeed appear to be 34919 bytes.  <strike>So, that seems to suggest sepolia is not following london exactly.  The only relevant EIP is [EIP3860](https://eips.ethereum.org/EIPS/eip-3860) which was introduced in Shanghai and increased the initcode size.  But, that doesn't make sense ...</strike>

@OlivierBBB points out that prior to EIP3860, there was not limit on initcode.  Therefore, on the london fork, we have unlimited initcode size.  That means the problem is with our replay environment which doesn't reflect this.

**Comment by DavePearce** [comment]
This should be resolved by #1143, noting however that it then immediately fails with another issue (#1145)
